### PR TITLE
chore(scripts/install): fix systemd check

### PR DESCRIPTION
**Pull request by sumo-drosiek**
Signed-off-by: Dominik Rosiek <drosiek@sumologic.com>

### CODE REVIEW

File: scripts/install.sh [hunk 1]
```diff
@@ -41,9 +41,11 @@ VERSION=""
 CONTINUE=false
 FILE_STORAGE=""
 CONFIG_DIRECTORY=""
-SYSTEMD_DISABLED=""
 SYSTEMD_CONFIG=""
 
+# set by dependency_check
```
Comment (sumo-drosiek) [inline scripts/install.sh:46]:
```suggestion
# set by check_dependencies
```

File: scripts/install.sh [hunk 2]
```diff
@@ -41,9 +41,11 @@ VERSION=""
 CONTINUE=false
 FILE_STORAGE=""
 CONFIG_DIRECTORY=""
-SYSTEMD_DISABLED=""
 SYSTEMD_CONFIG=""
 
+# set by check_dependencies
```
Comment (sumo-drosiek) [inline scripts/install.sh:46]:
```suggestion
# set by check_dependencies therefore cannot be set by set_defaults
```

File: scripts/install.sh [hunk 3]
```diff
@@ -41,9 +41,11 @@ VERSION=""
 CONTINUE=false
 FILE_STORAGE=""
 CONFIG_DIRECTORY=""
-SYSTEMD_DISABLED=""
 SYSTEMD_CONFIG=""
 
+# set by check_dependencies therefore cannot be set by set_defaults
+SYSTEMD_DISABLED=false
```
Comment (pdelewski) [inline scripts/install.sh:47]: Is any reason to move this variable to new line?

Comment (sumo-drosiek) [reply scripts/install.sh:47]: visual separation

**Pull request merged**